plotsquared/plot-flags.md

```diff
@@ -29,7 +29,7 @@ worlds:
 
 _`<world>` indicates the name of that plotworld._
 
-You can use [Vanilla block tags](vanilla-tags.md) too.
+You can use [vanilla block tags](vanilla-tags.md) too.
```

Comment:
Isn't Vanilla more like a proper name?

Reply:
I treat "vanilla" as an adjective due to where it's used "I like vanilla Minecraft," for instance. If you treat "Vanilla Minecraft" as the whole proper noun (I.E. including the "vanilla" into the proper noun) I'm fine with that interpretation too.